pandas/_libs/tslibs/np_datetime.pxd

```diff
@@ -120,3 +120,9 @@ cdef int64_t convert_reso(
     NPY_DATETIMEUNIT to_reso,
     bint round_ok,
 ) except? -1
+
+cdef extern from "src/datetime/np_datetime_strings.h":
+    cdef enum Exact:
```

Comment:
I think the name `Exact` is a little too vague - maybe better as `DatetimeFormatRequirement`?

Reply:
Yes good call, I've gone with `FormatRequirement` to keep lines not-too-long